src/component/registry/types.ts

```diff
@@ -54,4 +54,19 @@ export type OverlayPosition = OverlayShapePosition | OverlayEdgePosition;
 export interface Overlay {
   position: OverlayPosition;
   label?: string;
+  style?: {
+    font?: {
```

Comment:
❓ do we explain that the font family is the one used for the BPMN diagram label?

Reply:
Yes, I forgot to do the documentation. We should add it on the doc.